docs/modules/ROOT/pages/vshn-managed/keycloak/customization.adoc

```diff
@@ -0,0 +1,28 @@
+= Customization
+
+This page will describe how you can customize your Keycloak instance.
+
+== Themes and providers
+
+`VSHNKeycloak` allows to use custom providers and themes.
+xref:vshn-managed/how-tos/custom-image-keycloak.adoc[Create] a custom Docker image and use it to customize your Keycloak instance as described below.
```

Comment:
```suggestion
xref:vshn-managed/how-tos/custom-image-keycloak.adoc[Create] a custom container image and use it to customize your Keycloak instance as described below.
```

Just a nit, but I prefer to use the word "container" instead of Docker, as Docker is the name of the product...